assets/src/design-system/components/popup/utils/getOffset.js

```diff
@@ -106,12 +105,8 @@ export function getOffset(placement, spacing, anchor, dock, popup) {
   // Vertical
   const offsetY = getYOffset(placement, spacingV, anchorRect);
 
-  // use window.pageYOffset instead of bodyRect.height to account for scroll
   const maxOffsetY =
-    window.pageYOffset +
-    bodyRect.y -
-    height -
-    getYTransforms(placement) * height;
+    bodyRect.height + bodyRect.y - height - getYTransforms(placement) * height;
```

Comment:
@BrittanyIRL, can you check what exactly was the reason for switching to `window.pageYOffset` and just how can it replace `bodyRect.height` - for a non-scrolled window, the former is 0 but the latter is most certainly.

This fixes my issue, but might reintroduce issues in the dashboard. Could you please try to check this out locally and see if it recreates issues in the dashboard, please?

And if it does, can you see if you can fix that while not breaking this tooltip in the editor (see #6494 for example)?

Thanks!

Reply:
I was a little worries this might come up! I tested in storybook but that doesn't have the same structure as the editor. Thank you for catching and flagging this.

I think this splits the difference and I can live with it. I feel like this is just some weeds to get stuck in. 
This is what happens on this branch to a dropdown that requires scroll:
![updated popup results on dashboard](https://user-images.githubusercontent.com/10720454/109177751-5eb06780-7745-11eb-8940-cb5cab4880f6.gif)

I'd updated this to take into consideration the scroll height of the view but if it's causing problems for the editor by all means let's keep it how it was. I'd like to also draw attention to the other part of this file that I updated - line 114: 
`  const trueMaxOffsetY = maxOffsetY > 0 ? maxOffsetY : offsetY;` 

I kept seeing the maxOffsetY by less than 0 which was hijacking the popup to way higher on the page than it needs to be. I think this check combined with the older logic that you've restored keeps the scrolled into view dropdowns (below the fold) in the right place.

Reply:
(just updated the comment - some typos made the meaning a bit garbled and the ticket link was wrong)